api/src/test/java/com/epam/pipeline/manager/issue/AttachmentFileManagerTest.java

```diff
@@ -130,4 +134,18 @@ public void testDownloadAttachment() {
         verify(attachmentManager).load(1L);
         verify(dataStorageManager).getStreamingContent(testSystemDataStorage.getId(), TEST_ATTACHMENT_PATH, null);
     }
+
+    @Test
+    public void testDeleteAttachment() {
+        doReturn(true).when(authManager).isAdmin();
+
+        attachmentFileManager.deleteAttachment(anyLong());
+
+        verify(attachmentManager).load(anyLong());
+    }
```

Comment:
It looks like the test only checks if admin is able to delete the attachment. At the same time if we look at the original test it tests the behavior for an attachment owner. In order to be sure that we test the same functionality and even more we should cover the lost test scenario. 

Let's keep the test but add another one with attachment deletion for attachment owner.